src/api/services/ListingItemTemplateService.ts

```diff
@@ -419,6 +426,40 @@ export class ListingItemTemplateService {
         return messageSize;
     }
 
+    // sets an image as a "featured" image
```

Comment:
use proper js/java doc way to document functions
    /**
     * sets the featured image for the ListingItemTemplate
     *
     * @param listingItemTemplate
     * @param imageID
     */

 your ide should be able to quickly generate these. try "/**" +(command+enter) or something similar.